packages/hardhat/contracts/Ethernauts.sol

```diff
@@ -7,9 +7,8 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
-import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
```

Comment:
Since we're not using safeMint (and we never used safeTransfer), `checkOnERC721Received` will never be called, or any other post execution hook for that matter. Thus, I think it's safe to assume that there is no re-entrancy risk.